modules/react/skeleton/index.ts

```diff
@@ -1 +1 @@
-export * from './lib/skeleton';
+export {Skeleton} from './lib/skeleton';
```

Comment:
I limited the export here to `{Skeleton}` to compensate for the newly exported `SkeletonProps` in `skeleton.tsx`.